.github/workflows/ci.yaml

```
@@ -0,0 +1,28 @@
+name: ci
+
+on:
+  pull_request:
+  push:
+
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout source
+        uses: actions/checkout@v2
+
+      - name: python 3.8 setup
+        uses: actions/setup-python@v1
```

There is a newer version of the action out, let's use it:

```suggestion
        uses: actions/setup-python@v2
```